cron/cron.sh

```diff
@@ -9,3 +9,5 @@ do
     echo $line
     go run . --repo=$line --format=csv 2>/dev/null >> $output
 done < "$input"
+
+gsutil cp $output gs://ossf-scorecards/
```

Comment:
maybe gs://ossf-scorecards/ should come from ENV var, that ENV var can be set in Dockerfile,

ENV SCORECARS_BUCKET gs://ossf-scorecards

Reply:
handled this in the k8s yaml instead, it's basically the same thing